Source: Markus Rathgeb <maggu2810@web.de>
Upstream: Applied in trunk, 0.11 will be fixed.
Reason: avcodec_decode_audio is deprecated in latest ffmpeg.
http://bugs.gentoo.org/show_bug.cgi?id=246148
http://forums.musicbrainz.org/viewtopic.php?id=1369
--- picard-0.10/picard/musicdns/avcodec.c.org   2008-11-09 12:47:41.000000000 +0100
+++ picard-0.10/picard/musicdns/avcodec.c       2008-11-09 12:47:49.000000000 +0100
@@ -278,7 +278,8 @@
         data = packet.data;

         while (size > 0) {
-            len = avcodec_decode_audio(codec_context, (int16_t *)buffer_ptr, &output_size, data, size);
+            output_size = buffer_size + AVCODEC_MAX_AUDIO_FRAME_SIZE;
+            len = avcodec_decode_audio2(codec_context, (int16_t *)buffer_ptr, &output_size, data, size);

             if (len < 0)
                 break;
